internal/cmd/operator-sdk/generate/internal/genutil.go

```diff
@@ -34,6 +34,7 @@ import (
 
 // ValidateVersion returns an error if version is not a strict semantic version.
 func ValidateVersion(version string) error {
+	version = strings.TrimPrefix(version, "v")
 	v, err := semver.Parse(version)
 	if err != nil {
 		return fmt.Errorf("%s is not a valid semantic version: %v", version, err)
```

Comment:
Just a nit. It seems to be missing the entry in https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml.

Reply:
@cardil The changelog fragments get consumed and put [here](https://github.com/operator-framework/operator-sdk/tree/master/changelog/generated) each time we cut a release.

Reply:
Makes 100% sense. Although, this simple explanation should be stated in the docs.